src/heroes/bastion.opy

```diff
@@ -0,0 +1,216 @@
+playervar BastionTracker
+playervar BastionSentryAmmoBar
+playervar BastionTankMode
+playervar BastionSentryAmmo
+
+
+rule "Bastion - Technical Tracker":
+    @Event eachPlayer
+    @Condition eventPlayer.getCurrentHero() == Hero.BASTION
+    @Condition eventPlayer.hasStatusEffect(Status.HACKED) == false
+    
+    eventPlayer.BastionTracker = getLastCreatedText()
+    waitUntil(eventPlayer.getCurrentHero() != Hero.BASTION or eventPlayer.hasStatusEffect(Status.HACKED) == true, 99999)
+    destroyProgressBarHud(eventPlayer.BastionSentryAmmoBar)
+
+
+rule "Bastion - Continuous Sentry Mode":
+    @Event eachPlayer
+    @Hero bastion
+    @Condition eventPlayer.isUsingAbility1() == true
+    @Condition eventPlayer.BastionTankMode == false
+    @Condition eventPlayer.isDummy() == false
+    
+  
+    waitUntil(eventPlayer.isInAlternateForm() == false, 7.016)
```

Comment:
Where is this 7.016 from? Can you replace this with a keyword and define the value of the keyword in either "ow1_constants.opy' or "ow2_constants.opy" (aka avoid magic numbers)?

Reply:
That is how long bastion's overwatch 2 turret form is

Reply:
But yes, I will do that